addons/mail/static/src/components/message/message.js

```diff
@@ -274,6 +275,7 @@ class Message extends Component {
      * @returns {Object}
      */
     get trackingValues() {
+        const session = this.env.session;
```

Comment:
Ping @seb-odoo for JS part

Reply:
Don't bother creating a global const in this case, when it's just used once in a particular case.